webapi/_test_data/ontologies/beol-onto.ttl

```diff
@@ -483,7 +536,134 @@
 
                   rdfs:subPropertyOf knora-base:hasValue .
 
+    ### ###########################################
+    ### beol:hasFloruitDate
+
+    :hasFloruitDate rdf:type owl:ObjectProperty ;
+
+                  rdfs:label "Blühtedataum"@de ,
+                             "Floruit Date"@en ;
+
+                  rdfs:comment """Repräsentiert ein Blühtedataum."""@de,
```

Comment:
no according to martin, Floruit date is a date somebody was seen, or had a publication. in bibliography items this is indicated as (fl. 2.\,Hälfte 20.\,Jh.) This is used mostly for cases that no birth or death date of someone is available.